Framework/DataHandling/src/LoadNexusMonitors2.cpp

```diff
@@ -123,7 +123,8 @@ bool isEventMonitor(::NeXus::File &file) {
 //------------------------------------------------------------------------------
 /// Initialization method.
 void LoadNexusMonitors2::init() {
-  declareProperty(std::make_unique<API::FileProperty>("Filename", "", API::FileProperty::Load, ".nxs"),
+  const std::vector<std::string> exts{".nxs.h5", ".nxs", "_event.nxs"};
+  declareProperty(std::make_unique<API::FileProperty>("Filename", "", API::FileProperty::Load, exts),
                   "The name (including its full or relative path) of the NeXus file to "
                   "attempt to load. The file extension must either be .nxs or .NXS");
```

Comment:
this helpline doesn't agree with the allowed extensions `".nxs.h5", ".nxs", "_event.nxs"`. Can you update it?